tests/Application/Fees/CalculatorTest.php

```diff
@@ -73,10 +81,17 @@ public function testStripeUSCardSEKDonation(): void
         $this->assertEquals('5.74', $calculator->getCoreFee());
     }
 
-    public function testZeroVatChargedForUsaCharity()
```

Comment:
@alihejazi this was not testing what it claimed to before because the default VAT config for unit tests is all zeroed – see `phpunit.xml`.

To test the differing behaviour with realistic VAT configs now that we are live for VAT, you need to give the unit test different settings and have at least one test cover the UK vs. non-UK cases. I have now done that in `CalculatorTest` alongside my other changes.